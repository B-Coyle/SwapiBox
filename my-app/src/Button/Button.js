import React from 'react';
import './Button.scss';

export default function CardContainer(props) {
  return (
    <section className="mainBtns" onClick={event => props.selectCategory(event)} >
      <button className="button">people</button>
      <button className="button">planets</button>
      <button className="button">vehicles</button>
    </section>
  );
}
