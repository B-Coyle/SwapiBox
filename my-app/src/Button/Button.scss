.mainBtns{
    display: flex;
    justify-content: center;
}


.button{
    background-color: #36454f;
    color: #DBAe85;
    box-shadow: 5px 2px #b4b4b4;
    border-radius: 5px;
    border: 1px solid;
    width: 200px;
    margin: 20px;
    height: 100px;
    cursor: pointer;
    
}

.button:hover{
    background-color: #061283;
    color: #fff
}
