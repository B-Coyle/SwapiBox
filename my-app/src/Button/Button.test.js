import React from 'react';
import Button from '../Button/Button.js';
import {shallow} from 'enzyme';
import { isMainThread } from 'worker_threads';

describe('Button', () => {
    let wrapper

it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
    });

})
