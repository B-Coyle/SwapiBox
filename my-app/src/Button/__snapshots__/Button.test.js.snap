// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot with all data passed in 1`] = `undefined`;
